sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -551,7 +551,7 @@ export interface RetryOptions {
 
 // @public
 export interface SasTokenProvider {
-    getToken(audience: string): AccessToken;
+    getToken(audience: string): Promise<AccessToken>;
```

Comment:
feels weird that this wasn't async to begin with given getToken() is always async in other credentials